core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -235,11 +235,11 @@
   MASTER_RECOVERY_DELAY("master.recovery.delay", "10s", PropertyType.TIMEDURATION,
       "When a tablet server's lock is deleted, it takes time for it to completely quit. This delay gives it time before log recoveries begin."),
   MASTER_LEASE_RECOVERY_WAITING_PERIOD("master.lease.recovery.interval", "5s", PropertyType.TIMEDURATION,
-      "The amount of time to wait after requesting a WAL file to be recovered"),
+      "The amount of time to wait after requesting a write-ahead log to be recovered"),
   MASTER_WALOG_CLOSER_IMPLEMETATION("master.walog.closer.implementation", "org.apache.accumulo.server.master.recovery.HadoopLogCloser", PropertyType.CLASSNAME,
-      "A class that implements a mechansim to steal write access to a file"),
+      "A class that implements a mechanism to steal write access to a write-ahead log"),
   MASTER_FATE_THREADPOOL_SIZE("master.fate.threadpool.size", "4", PropertyType.COUNT,
-      "The number of threads used to run FAult-Tolerant Executions. These are primarily table operations like merge."),
+      "The number of threads used to run Fault-Tolerant Executions. These are primarily table operations like merge."),
```

Comment:
The capitalization wasn't a typo. It was highlighting where the name "FATE" comes from. If we're not going to highlight that, then none of it should be capitalized.

Reply:
I didn't realize that. I can change it back but maybe we should be clear and use `fault-tolerant executions (i.e FATE)`